x-pack/legacy/plugins/siem/public/components/drag_and_drop/draggable_wrapper_hover_content.tsx

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiButtonIcon, EuiToolTip } from '@elastic/eui';
+import React, { useCallback, useMemo } from 'react';
+
+import { getAllFieldsByName, WithSource } from '../../containers/source';
+import { WithCopyToClipboard } from '../../lib/clipboard/with_copy_to_clipboard';
+import { useKibana } from '../../lib/kibana';
+import { createFilter } from '../page/add_filter_to_global_search_bar';
+import { useTimelineContext } from '../timeline/timeline_context';
+import { StatefulTopN } from '../top_n';
+
+import { allowTopN } from './helpers';
+import * as i18n from './translations';
+
+interface Props {
+  field: string;
+  onFilterAdded?: () => void;
+  showTopN: boolean;
+  toggleTopN: () => void;
+  value?: string[] | string | null;
+}
+
+const DraggableWrapperHoverContentComponent: React.FC<Props> = ({
+  field,
+  onFilterAdded,
+  showTopN,
+  toggleTopN,
+  value,
+}) => {
+  const kibana = useKibana();
+  const { filterManager: timelineFilterManager } = useTimelineContext();
+  const filterManager = useMemo(() => kibana.services.data.query.filterManager, [
+    kibana.services.data.query.filterManager,
+  ]);
+
+  const filterForValue = useCallback(() => {
```

Comment:
Instead of using filterForValue and filterOutValue, I think we can do something like that to avoid this two functions to be really similar
```
const filterValue = useCallback(
    (negateOptions: boolean) => {
      const negate = value?.length === 0 ? false : negateOptions;
      const filter =
        value?.length === 0
          ? createFilter(field, null, negate)
          : createFilter(field, value, negate);
      const activeFilterManager = timelineFilterManager ?? filterManager;

      if (activeFilterManager != null) {
        activeFilterManager.addFilters(filter);

        if (onFilterAdded != null) {
          onFilterAdded();
        }
      }
    },
    [field, value, timelineFilterManager, filterManager, onFilterAdded]
  );
```
 #### Filter For
```
<EuiButtonIcon
            aria-label={i18n.FILTER_FOR_VALUE}
            color="text"
            data-test-subj="filter-for-value"
            iconType="magnifyWithPlus"
            onClick={filterValue.bind(null, false)}
          />
```
#### Filter Out
```
<EuiButtonIcon
            aria-label={i18n.FILTER_OUT_VALUE}
            color="text"
            data-test-subj="filter-out-value"
            iconType="magnifyWithMinus"
            onClick={filterValue.bind(null, true)}
          />
```